### PR TITLE
App central incident tool integration

### PR DESCRIPTION
**Pull request by rishav-sumo-dev**
## Purpose of this pull request

This pull request... 

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: docs/platform-services/automation-service/app-central/integrations/servicenow-v2.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ---
 title: ServiceNow V2
 description: ''
+tags: []
```
Comment (kimsauce) [inline docs/platform-services/automation-service/app-central/integrations/servicenow-v2.md:4]: We're no longer inserting this `tags` line in metadata

```suggestion
```

File: docs/platform-services/automation-service/app-central/integrations/incident-tools.md [hunk 2]
```diff
@@ -85,3 +85,4 @@ Supervised Active Intelligence Automation
     * Search Entities
     * Search Incidents
     * Update Entity
+* June 24, 2024 (v1.8) - Removed the requirement for ***Field ID*** and ***Field Value*** fields in the **Update Incident Fields** action (the fields are now optional).
```
Comment (kimsauce) [inline docs/platform-services/automation-service/app-central/integrations/incident-tools.md:88]:
```suggestion
* June 24, 2024 (v1.8) - The **Field ID** and **Field Value** fields are now optional in the **Update Incident Fields** action.
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
one suggestion

**Review by JV0812 — APPROVED** [review]
LGTM

**Review by kimsauce — APPROVED** [review]

**Pull request merged**